ycms/cms/views/ward/ward_management_view.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib import messages
+from django.shortcuts import redirect, render
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...constants import job_types
+from ...forms import WardForm
+from ...models import User, Ward
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+
+    template_name = "ward/ward_management.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        This function returns a list of all wards,
+        as well as a form for creating a new one
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+
+        """
+
+        all_wards = Ward.objects.all()
+        ward_form = WardForm()
+
+        return render(
+            request,
+            self.template_name,
+            {
+                "all_wards": all_wards,
+                "wards": self._get_ward_info(all_wards),
+                "ward_form": ward_form,
+                # **super().get_context_data(**kwargs),
+            },
+        )
```

Comment:
```suggestion
        return render(
            request,
            self.template_name,
            {
                "ward_form": ward_form,
                **self._get_ward_info(),
                **super().get_context_data(**kwargs),
            },
        )
```
This will "unpack" the dictionary you get from `self._get_ward_info` - the resulting dict that will be returned contains all the same `"key": value` pairs as the function returns.

I would also move the fetching of the wards objects to that new method.

Also please keep the `super().get_context_data`, depending on our request handling / routing / the base class the view is inheriting from, this actually does a lot.